### ISSUE TITLE
SharedStorageTest::testGetPermissions fails

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#4683]
```
There was 1 failure:
1) OCA\Files_Sharing\Tests\SharedStorageTest::testGetPermissions
Failed asserting that 11 is identical to 9.
/drone/src/github.com/nextcloud/server/apps/files_sharing/tests/SharedStorageTest.php:213
```

@icewind1991 @schiessle @rullzer Do you have an idea why this test is failing? I try to bisect this.

**Comment by MorrisJobke** [comment]
Broken by #4623 - let me revert it.

**Comment by MorrisJobke** [comment]
Fixed via #4684

**Comment by rullzer** [comment]
:see_no_evil: that test is just wrong... we should have permissions to update it... since you can rename a share mountpoint... will take care of pr with fix...
